.pep8speaks.yml

```diff
@@ -0,0 +1,23 @@
+# File : .pep8speaks.yml
+
+message:  # Customize the comment made by the bot
+    opened:  # Messages when a new PR is submitted
+        header: "Hello @{name}, Thank you for submitting the Pull Request !"
+                # The keyword {name} is converted into the author's username
+        footer: "If you have not done so, please consult the [`jwql` Style Guide](https://github.com/spacetelescope/jwql/blob/master/style_guide/style_guide.md)"
+                # The messages can be written as they would over GitHub
+    updated:  # Messages when new commits are added to the PR
+        header: "Hello @{name}, Thank you for updating !"
+        footer: ""  # Why to comment the link to the style guide everytime? :)
+    no_errors: "Cheers ! There are no PEP8 issues in this Pull Request. :beers: "
+
+scanner:
+    diff_only: False  # If True, errors caused by only the patch are shown
+
+pycodestyle:
+    max-line-length: 200  # Default is 79 in PEP8
```

Comment:
200 seems a little long... I understand occasionally going over the line limit by 10 or so characters.  Or was there something already in our code you had in mind with this value?  I'd say something like 100 (120?) characters should be our max?

Reply:
@SaOgaz Good point.  I'll set it to something more reasonable.